### ISSUE TITLE
Cannot create new users error

### ISSUE THREAD

**Issue by spfandanf** [uglyDwarf/linuxtrack#115]
I'm new to linuxtrack and trying to use it with trackir and xplane.  I tried to install it on my imac using the ltr.gui file but it kept giving me a "user could not be created" error.  I'm not very savvy when it comes to computer programming, so you'll have to forgive me for not giving too many details.  Any help would be great, thank you!!

**Comment by uglyDwarf** [comment]
Hi,
just to make sure we are on the same page... Download the newest linuxtrack dmg from here: https://github.com/uglyDwarf/linuxtrack/issues/113 , open it and drag the ltr_gui to your desktop...
When it finished copying, try to start it up and should any error or warning pop up, please either copy it here verbatim or do the screenshot...
Unfortunately the error message doesn't make much sense, since the ltr_gui doesn't create any new users...
Kind regards,
Michal

PS. Which MacOS X version are you on?

**Comment by spfandanf** [comment]
Yes, I just tried again with that same link.  When I copy it to the desktop and start it I get an error that reads, "Can't create '/Users/The Fitzgeralds/.config/linuxtrack'!".  That is copied verbatim.  I am using OS X El Capitan version 10.11.3.  Thank you for any help you can provide!

**Comment by uglyDwarf** [comment]
Hi,
the error message is a complaint, that the directory to store the setup files for Linuxtrack couldn't be created. There are two possibilities - number one, there is a permissions problem; number two, the username with a space inside of it is causing the problem...
What you can try is this: open up a terminal application (in Applications/Utilities) and copy/paste there the following command:
`mkdir -p '/Users/The Fitzgeralds/.config/linuxtrack'
`
If that passes without an error message, then the culprit is option number two; if you also get an error message, please copy it here, so I can see what is wrong...
Kind regards,
Michal

**Comment by spfandanf** [comment]
It then gives me this error inside the terminal box: mkdir: /Users/The Fitzgeralds/.config/linuxtrack: Permission denied
Seans-iMac-4:~ The Fitzgeralds$ 
So that would be a permissions problem, I'm assuming?  How would I fix that? 
Thanks again!

**Comment by uglyDwarf** [comment]
Hi,
it looks like the permissions problem, the question is on which level is it...
You can try these commands:
chmod u+rwx .
chmod u+rwx .config

The second one might fail if the .config file doesn't exist; anyway, after running those two commands, the the mkdir command from the above post - if it doesn't throw an error again, try the ltr_gui...
Kind regards,
Michal
